installing/installing_gcp/installing-restricted-networks-gcp.adoc

```diff
@@ -0,0 +1,106 @@
+[id="installing-restricted-networks-gcp"]
+= Installing a cluster on GCP in a restricted network
+include::modules/common-attributes.adoc[]
+:context: installing-restricted-networks-gcp
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Google Cloud Platform (GCP) by using infrastructure that you provide and an internal mirror of the installation release content.
```

Comment:
s/by using/using ?

Reply:
> ! using n
To avoid ambiguity, replace this gerund with either “by using” or “that uses.”

Reply:
In that case, would `that uses` be better?